rosa/rosa-wrapper.py

```diff
@@ -105,6 +104,28 @@ def _install_addons(rosa_cmnd,cluster_id,addons):
             logging.error(addon_stderr.strip().decode("utf-8"))
         # TODO: control addon is installed with: rosa list addons -c <<cluster_name>>
 
+def _add_machinepool(rosa_cmnd,cluster_id):
+    logging.info('Creating machinepool %s on %s' % (args.machinepool_name,cluster_id))
+#  rosa create machinepool -c mycluster --name=mp-1 --replicas=2 --instance-type=r5.2xlarge --labels =foo=bar,bar=baz"
+    machinepool_cmd = [rosa_cmnd, "create", "machinepool",
+                       "--cluster", cluster_id,
+                       "--instance-type", args.machinepool_flavour,
+                       "--name", args.machinepool_name,
+                       "--labels", args.machinepool_labels,
+                       "--taints", args.machinepool_taints,
+                       "--replicas", args.machinepool_replicas,
+                       "-y"]
+    logging.debug(machinepool_cmd)
+    machinepool_process = subprocess.Popen(machinepool_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    machinepool_stdout,machinepool_stderr = machinepool_process.communicate()
+    if machinepool_process.returncode != 0:
+        logging.error('Unable to create machinepool %s on %s' % (args.machinepool_name,cluster_id))
+        logging.error(machinepool_stdout.strip().decode("utf-8"))
+        logging.error(machinepool_stderr.strip().decode("utf-8"))
+    else:
+        logging.info('Created machinepool %s on %s. Waiting 5 minutes for hosts to come up' % (args.machinepool_name,cluster_id))
+        time.sleep(300)
```

Comment:
Same comment as the osde2e one. -1 on the blind sleep

Reply:
added active waiting for machinepool creation